docs/content/development/extensions.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: doc_page
+---
+
+# Core extensions
+
+Druid comes with a number of core extensions to enhance functionality. These extensions can be enabled in the `druid.extensions.loadList` config. 
+For more information, please see [here](../operations/including-extensions.html).
+
+Core extensions include support for:
+
```

Comment:
This list is probably going to be confusing to someone not familiar with Druid:

- it's not clear what the extensions actually do (is "Avro" a data storage format or an input parser? is HDFS a deep storage or an input connector? what is Kafka used for and why is it listed three times?)
- some of them never have their names listed anywhere (you can't load the Kafka namespace lookup thing unless you know the magic words `druid-kafka-extraction-namespace`

So I think it'd be better, for each of these extensions, to list:

- what they do: are they a deep store / metadata store / aggregator / firehose / lookup / input parser
- what their actual loadable name is
- links to relevant configurations (deep store / metadata stores need properties; aggregators / firehoses / lookups / input parsers need specs)